### DISCUSSION TITLE
EDPStatic_W/R_ALIVE Property are being generated and occupying traffic.

### DISCUSSION THREAD

**Discussion by Herbifors** [eProsima/Fast-DDS#3344] [Q&A]
## Problum : When configuring a static environment, many EDPStatic_W/R_ALIVE Property are being generated and occupying traffic.
-----
Wireshark Capture : [Wireshark_Capture.zip](https://github.com/eProsima/Fast-DDS/files/10897541/Wireshark_Capture.zip)
-----
While configuring the static environment, I noticed that 18 properties named EDPStatic_W/R_ALIVE were created.
and I confirmed that one Property was due to what I set, but I don't know why the remaining 17 EDPStatic_W/R_ALIVEs were created.
This EDPStatic_W/R_ALIVE Property is taking up a lot of traffic, 
I wonder if it can be removed and curious about the role of the EDPStatic_W/R_ALIVE Property.

**Comment by JLBuenoLopez** [comment]
Hi @Herbifors 

Even though the Endpoint Discovery Phase (EDP) is static, the remote DomainParticipant has to let know the local DomainParticipant which endpoints have been created. If you look at the value of the property, each one of them is indicating a Writer o Reader alive and the Entity ID that has been set in the Static EDP XML configuration file.

You might be interested in the [following configuration](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/non_consolidated_qos.html#static-discovery-s-exchange-format) in order to reduce the bandwidth of those properties.

**Reply by Herbifors** [reply]
Thanks. That was helpful. However, is it possible to remove the 17 EDS_W/R_65535 properties, except for the EDS_RA_103 I set, from the PID_PROPERTY_LIST? Or Are the 17 EDS_W/R_65535 properties essential elements for configuring the static environment?
I want to minimize the traffic as much as possible.
Changed Wireshark Capture : [Reduced.zip](https://github.com/eProsima/Fast-DDS/files/10904322/Reduced.zip)

**Reply by JLBuenoLopez** [reply]
I am afraid that those properties are required in the Fast DDS Static EDP discovery mechanism. As no EDP discovery traffic is going through, the Participant should let know to the remote Participant which endpoints have been created. This discovery mechanism allows for defining potential endpoints that finally are not enabled into the communication. The only current manner to let know which endpoints should be matched/unmatched remotely is using those properties.
